lib/main.js

```diff
@@ -69,11 +69,10 @@ var describeModule = function (module, describe, it) {
       it(test.test, function () {
         forEach(test.assertions, function (assertion) {
           if (!assertion.passed) {
-            assert.fail(
+            assert.strictEqual(
               assertion.output,
               assertion.expected,
-              formatFailureMessage(assertion),
-              assertion.assertionType
+              formatFailureMessage(assertion)
```

Comment:
This assumes that the comparison is always `strictEqual`, which will cause any other assertion (e.g. `unequal`) to report as successful when it actually failed.

I think what we actually want here is `assert.fail(formatFailureMessage(assertion))`, but that loses the nice colored logging of `expected` and `actual`... so we probably also want to expand the failure message a bit? I'll submit a new PR.